lib/WiiExtension/WiiExtension.h

```diff
@@ -61,12 +71,16 @@
 #endif
 
 #ifndef WII_EXTENSION_CALIBRATION
```

Comment:
Same with this. If we wanted to enable/disable extension calibration, we should make a web config checkbox. Otherwise we can assume we always want it :)

Reply:
I want to leave this as is since not all devices currently support on-device calibration.  The fallback should be enough for _most_ cases, but we may need further testing on third party to know for certain.